### ISSUE TITLE
Website icons not showing up

### ISSUE THREAD

**Issue by hrvoje-hr** [FriendsOfFlarum/socialprofile#27]
## Bug Report

**Current Behavior**
External website icons are not displayed in social icons field

**Steps to Reproduce**
Go to profile page, enter website link in the social icons field

**Expected Behavior**
External website icon should be displayed

**Screenshots**
https://i.ibb.co/NtG1qVj/Schermafbeelding-2021-11-02-230511.png

**Environment**
- Flarum version: 1.1.1
- Extension version: 1.1.0
- Hosting environment: VPS
- Browser: Chrome, Firefox, Opera latest versions

```
| flarum-flags                  | v1.1.0  |        |
| flarum-tags                   | v1.1.0  |        |
| flarum-subscriptions          | v1.1.0  |        |
| flarum-approval               | v1.1.0  |        |
| flarum-suspend                | v1.1.0  |        |
| flarum-markdown               | v1.1.0  |        |
| the-turk-nodp                 | 1.0.1   |        |
| sycho-profile-cover           | v1.3.0  |        |
| sycho-move-posts              | v0.1.5  |        |
| fof-socialprofile             | 1.1.0   |        |
| fof-reactions                 | 1.0.2   |        |
| fof-links                     | 1.0.3   |        |
| fof-gamification              | 1.2.0   |        |
| fof-formatting                | 1.0.1   |        |
| fof-follow-tags               | 1.0.2   |        |
| fof-drafts                    | 1.0.3   |        |
| fof-byobu                     | 1.0.1   |        |
| flarum-sticky                 | v1.1.0  |        |
| flarum-statistics             | v1.1.0  |        |
| flarum-nicknames              | v1.1.0  |        |
| flarum-mentions               | v1.1.1  |        |
| flarum-lock                   | v1.1.0  |        |
| flarum-lang-english           | v1.1.0  |        |
| flarum-emoji                  | v1.1.0  |        |
| clarkwinkelmann-author-change | 1.0.2   |        |
| askvortsov-rich-text          | v2.1.0  |   
```

**Additional Context**

In **edit buttons** menu favicons are loaded as they should, but not in the social icons field.

Also reported by other users on Disucss.

**Comment by dsevillamartin** [comment]
@hrvoje-hr In case this is still an issue, did you perhaps have the option disabled in settings? It was added in the v1 update

![image](https://user-images.githubusercontent.com/6401250/145127139-c58a81b8-2b04-43a0-ae9a-1f1e03718ad1.png)

**Comment by Xwdit** [comment]
> @hrvoje-hr In case this is still an issue, did you perhaps have the option disabled in settings? It was added in the v1 update
> 
> ![image](https://user-images.githubusercontent.com/6401250/145127139-c58a81b8-2b04-43a0-ae9a-1f1e03718ad1.png)

I have this problem too, allow external website icons already enabled

**Comment by imorland** [comment]
Testing this using Firefox, it seems the external icons are blocked:

![image](https://user-images.githubusercontent.com/16573496/172561711-8a834aa7-f892-4773-8e17-64b8f32f6514.png)

I've not yet tried other browsers

**Comment by dsevillamartin** [comment]
Well, Privacy Badger is an extension. It makes sense for it to be blocked due to potential tracking, I guess. I don't think we should attempt to solve this problem, perhaps just add some error handling to the icons instead and/or default to web icon.

**Comment by davwheat** [comment]
This might be fixed by 32c0aebbefd00a643b53290d4438259510a18b97
